sirepo/template/silas.py

```diff
@@ -425,53 +436,49 @@ def _z_label(plot_type):
             excited_states="Number [1/m³]",
         )[plot_type]
 
-    def _x_label(plot_type):
-        return PKDict(
-            excited_states_longitudinal="Crystal Slice",
-            longitudinal_photons="Crystal width [cm]",
-            longitudinal_intensity="Pulse Slice",
-        )[plot_type]
-
-    def _index(index, plot_type):
-        if plot_type == "longitudinal_photons":
-            return index
-        return index + 1
-
-    with h5py.File(run_dir.join(_fname(element).format(element_index)), "r") as f:
-        if _is_longitudinal_plot(plot_type):
-            x = []
-            y = []
-            nslice = _nslice(element, f)
-            if element:
-                element.nslice = nslice
-            for idx in range(nslice):
-                x.append(_index(idx, plot_type))
-                y.append(_y_value(element, idx, f, _cell_volume(element)))
-            return template_common.parameter_plot(
-                x,
-                [
-                    PKDict(
-                        points=y,
-                        label=_label(plot_type, slice_index),
-                    ),
-                ],
-                PKDict(),
-                PKDict(
-                    x_label=_x_label(plot_type),
-                ),
-            )
-        d = template_common.h5_to_dict(f, str(slice_index))
-        r = d.ranges
-        z = d[plot_type]
-        return PKDict(
-            title=_title(plot_type, slice_index),
-            x_range=[r.x[0], r.x[1], len(z)],
-            y_range=[r.y[0], r.y[1], len(z[0])],
-            x_label="Horizontal Position [m]",
-            y_label="Vertical Position [m]",
-            z_label=_z_label(plot_type),
-            z_matrix=z,
-        )
+    tries = 3
```

Comment:
doesn't seem like needs `tries` var

Reply:
But maybe could be constant

Reply:
I moved it to the top of the file as _MAX_H5_READ_TRIES = 3